packages/admin/tests/index.html

```diff
@@ -18,6 +18,13 @@
   <body>
     {{content-for "body"}} {{content-for "test-body"}}
 
+    <div id="qunit"></div>
```

Comment:
Is this updating the admin package too?

Reply:
My admin and data builds are filing. I was trying a change to see if it would pass